ChatQnA/ui/svelte/playwright.config.ts

```diff
@@ -38,7 +38,7 @@ export default defineConfig({
 		/* Maximum time each action such as `click()` can take. Defaults to 0 (no limit). */
 		actionTimeout: 0,
 		/* Base URL to use in actions like `await page.goto('/')`. */
-		baseURL: "http://localhost:80",
+		baseURL: "http://localhost:5173",
```

Comment:
Please revert this update.

Reply:
We have fix the nginx issue in GenAIComps, please revert this change and the CI on gaudi/xeon should able to pass.

Reply:
Need to keep 80 here.